### PR TITLE
chore: Reducing bundle size of Stack by moving selector used in multiple places to local const

### PR DESCRIPTION
**Pull request by khmakoto**
## PR Description

This PR moves the children selector logic in `Stack`'s styles to a local const that is referenced to avoid duplication and help reduce bundle size of the component.

## Related Issue(s)

Applies suggestions from [this comment](https://github.com/microsoft/fluentui/pull/25397#discussion_r1007459264) in #25397

### CODE REVIEW

**Review by sopranopillow — APPROVED** [review]

**Comment by fabricteam** [comment]
## 📊 Bundle size report

<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/6c136eb60ae73382d3336b2ff11b113924d6eddc'>6c136eb60ae73382d3336b2ff11b113924d6eddc</a></sub>
<div id="bundleSizeComment9423"/>

**Review by spmonahan — APPROVED** [review]

File: packages/react/src/components/Stack/Stack.styles.ts [hunk 1]
```diff
@@ -47,9 +47,11 @@ export const styles: IStackComponent['styles'] = (props, theme, tokens): IStackS
     textOverflow: 'ellipsis',
   };
 
+  const childSelector = enableScopedSelectors ? '.' + GlobalClassNames.child : '*';
```
Comment (behowell) [inline packages/react/src/components/Stack/Stack.styles.ts:50]: nit: technically `>` is part of the child selector, and always precedes `${childSelector}`. If you include it in the const:
```suggestion
  const childSelector = enableScopedSelectors ? '> .' + GlobalClassNames.child : '> *';
```

Then you can simplify below:
```diff
-          [`> ${childSelector}`]: {
+          [childSelector]: {
```

**Review by behowell — APPROVED** [review]

File: packages/react/src/components/Stack/Stack.styles.ts (hunk 1 above)
Comment (khmakoto) [reply packages/react/src/components/Stack/Stack.styles.ts]: Done!

**Comment by fabricteam** [comment]
## [Perf Analysis (`@fluentui/react`)](https://github.com/microsoft/fluentui/wiki/Perf-Testing)

  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">Master Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>TeachingBubble</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25429/perf-test/TeachingBubble-mount_base.html">84532</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25429/perf-test/TeachingBubble-mount.html">84943</a></td>
            <td>5000</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25429/perf-test/TeachingBubble-mount.regression.txt">Possible regression</a></td>
           </tr></table><details><summary>All results</summary><p>
  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">Master Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>BaseButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25429/perf-test/BaseButton-mount_base.html">1199</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25429/perf-test/BaseButton-mount.html">1177</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Breadcrumb</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25429/perf-test/Breadcrumb-mount_base.html">2950</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25429/perf-test/Breadcrumb-mount.html">2948</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Checkbox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25429/perf-test/Checkbox-mount_base.html">2635</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25429/perf-test/Checkbox-mount.html">2625</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>CheckboxBase</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25429/perf-test/CheckboxBase-mount_base.html">2364</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25429/perf-test/CheckboxBase-mount.html">2338</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ChoiceGroup</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25429/perf-test/ChoiceGroup-mount_base.html">4404</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25429/perf-test/ChoiceGroup-mount.html">4393</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ComboBox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25429/perf-test/ComboBox-mount_base.html">1215</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25429/perf-test/ComboBox-mount.html">1245</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>CommandBar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25429/perf-test/CommandBar-mount_base.html">9947</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25429/perf-test/CommandBar-mount.html">9576</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>ContextualMenu</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25429/perf-test/ContextualMenu-mount_base.html">11096</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25429/perf-test/ContextualMenu-mount.html">11072</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>DefaultButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25429/perf-test/DefaultButton-mount_base.html">1364</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25429/perf-test/DefaultButton-mount.html">1373</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRow</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25429/perf-test/DetailsRow-mount_base.html">3626</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25429/perf-test/DetailsRow-mount.html">3608</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRowFast</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25429/perf-test/DetailsRowFast-mount_base.html">3597</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25429/perf-test/DetailsRowFast-mount.html">3606</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRowNoStyles</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25429/perf-test/DetailsRowNoStyles-mount_base.html">3468</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25429/perf-test/DetailsRowNoStyles-mount.html">3467</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Dialog</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25429/perf-test/Dialog-mount_base.html">3077</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25429/perf-test/Dialog-mount.html">3063</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>DocumentCardTitle</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25429/perf-test/DocumentCardTitle-mount_base.html">576</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25429/perf-test/DocumentCardTitle-mount.html">584</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Dropdown</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25429/perf-test/Dropdown-mount_base.html">3180</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25429/perf-test/Dropdown-mount.html">3261</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FocusTrapZone</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25429/perf-test/FocusTrapZone-mount_base.html">2006</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25429/perf-test/FocusTrapZone-mount.html">2015</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FocusZone</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25429/perf-test/FocusZone-mount_base.html">1913</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25429/perf-test/FocusZone-mount.html">1943</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedList</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25429/perf-test/GroupedList-mount_base.html">52331</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25429/perf-test/GroupedList-mount.html">59456</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedList</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25429/perf-test/GroupedList-virtual-rerender_base.html">24994</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25429/perf-test/GroupedList-virtual-rerender.html">24772</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedList</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25429/perf-test/GroupedList-virtual-rerender-with-unmount_base.html">92093</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25429/perf-test/GroupedList-virtual-rerender-with-unmount.html">93791</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedListV2</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25429/perf-test/GroupedListV2-mount_base.html">550</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25429/perf-test/GroupedListV2-mount.html">558</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedListV2</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25429/perf-test/GroupedListV2-virtual-rerender_base.html">544</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25429/perf-test/GroupedListV2-virtual-rerender.html">551</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedListV2</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25429/perf-test/GroupedListV2-virtual-rerender-with-unmount_base.html">560</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25429/perf-test/GroupedListV2-virtual-rerender-with-unmount.html">572</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>IconButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25429/perf-test/IconButton-mount_base.html">1916</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25429/perf-test/IconButton-mount.html">1899</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Label</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25429/perf-test/Label-mount_base.html">727</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25429/perf-test/Label-mount.html">739</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Layer</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25429/perf-test/Layer-mount_base.html">4215</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25429/perf-test/Layer-mount.html">4233</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Link</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25429/perf-test/Link-mount_base.html">825</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25429/perf-test/Link-mount.html">831</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>MenuButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25429/perf-test/MenuButton-mount_base.html">1668</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25429/perf-test/MenuButton-mount.html">1641</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>MessageBar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25429/perf-test/MessageBar-mount_base.html">2337</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25429/perf-test/MessageBar-mount.html">2291</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Nav</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25429/perf-test/Nav-mount_base.html">3267</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25429/perf-test/Nav-mount.html">3221</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>OverflowSet</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25429/perf-test/OverflowSet-mount_base.html">1392</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25429/perf-test/OverflowSet-mount.html">1342</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Panel</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25429/perf-test/Panel-mount_base.html">2518</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25429/perf-test/Panel-mount.html">2523</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Persona</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25429/perf-test/Persona-mount_base.html">1260</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25429/perf-test/Persona-mount.html">1262</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Pivot</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25429/perf-test/Pivot-mount_base.html">1635</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25429/perf-test/Pivot-mount.html">1654</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>PrimaryButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25429/perf-test/PrimaryButton-mount_base.html">1539</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25429/perf-test/PrimaryButton-mount.html">1536</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Rating</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25429/perf-test/Rating-mount_base.html">7037</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25429/perf-test/Rating-mount.html">6995</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SearchBox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25429/perf-test/SearchBox-mount_base.html">1482</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25429/perf-test/SearchBox-mount.html">1509</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Shimmer</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25429/perf-test/Shimmer-mount_base.html">2836</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25429/perf-test/Shimmer-mount.html">2840</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Slider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25429/perf-test/Slider-mount_base.html">2123</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25429/perf-test/Slider-mount.html">2074</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SpinButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25429/perf-test/SpinButton-mount_base.html">4688</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25429/perf-test/SpinButton-mount.html">4685</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Spinner</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25429/perf-test/Spinner-mount_base.html">803</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25429/perf-test/Spinner-mount.html">803</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SplitButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25429/perf-test/SplitButton-mount_base.html">3069</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25429/perf-test/SplitButton-mount.html">3100</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Stack</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25429/perf-test/Stack-mount_base.html">828</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25429/perf-test/Stack-mount.html">830</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>StackWithIntrinsicChildren</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25429/perf-test/StackWithIntrinsicChildren-mount_base.html">2463</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25429/perf-test/StackWithIntrinsicChildren-mount.html">2446</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>StackWithTextChildren</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25429/perf-test/StackWithTextChildren-mount_base.html">4920</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25429/perf-test/StackWithTextChildren-mount.html">4906</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SwatchColorPicker</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25429/perf-test/SwatchColorPicker-mount_base.html">10497</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25429/perf-test/SwatchColorPicker-mount.html">11260</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TagPicker</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25429/perf-test/TagPicker-mount_base.html">2634</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25429/perf-test/TagPicker-mount.html">2662</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TeachingBubble</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25429/perf-test/TeachingBubble-mount_base.html">84532</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25429/perf-test/TeachingBubble-mount.html">84943</a></td>
            <td>5000</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25429/perf-test/TeachingBubble-mount.regression.txt">Possible regression</a></td>
           </tr>
<tr>
            <td>Text</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25429/perf-test/Text-mount_base.html">787</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25429/perf-test/Text-mount.html">783</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TextField</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25429/perf-test/TextField-mount_base.html">1620</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25429/perf-test/TextField-mount.html">1583</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25429/perf-test/ThemeProvider-mount_base.html">1507</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25429/perf-test/ThemeProvider-mount.html">1522</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25429/perf-test/ThemeProvider-virtual-rerender_base.html">1082</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25429/perf-test/ThemeProvider-virtual-rerender.html">1085</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25429/perf-test/ThemeProvider-virtual-rerender-with-unmount_base.html">2140</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25429/perf-test/ThemeProvider-virtual-rerender-with-unmount.html">2161</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Toggle</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25429/perf-test/Toggle-mount_base.html">1085</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25429/perf-test/Toggle-mount.html">1133</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>buttonNative</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25429/perf-test/buttonNative-mount_base.html">537</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25429/perf-test/buttonNative-mount.html">540</a></td>
            <td>5000</td>
            <td></td>
           </tr></table></p></details>

<div id="perfComment9423"/>

**Pull request merged**